styles/global.css

```diff
@@ -110,15 +105,21 @@ body {
 .dropdown:hover .dropbtn {
   color: grey;
 }
-
+.dropdown {
+  /* father of dropdown-content (positioned accord to father) */
+  position: relative;
+}
 .dropdown-content {
   background-color: black;
   /* hide dropdown content by default */
   display: none;
-  width: 100px;
+  width: 100%;
   /* make content appear below dropdown button */
   position: absolute;
   top: 100%;
+  left: 0;
+  /* make sure padding and border are included in total width */
+  box-sizing: border-box;
```

Comment:
box-sizing should be applied globally. Perhaps something like this in global.css
```
html * {
  box-sizing: border-box;
}
```